rust/Cargo.toml

```diff
@@ -29,8 +29,8 @@ lazy_static = "1"
 
 # Azure
 reqwest = { version = "0", optional = true }
-azure_core = { git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev = "536da42ebefd411feff8ba6a0965865e2741267e" }
-azure_storage = { git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev = "536da42ebefd411feff8ba6a0965865e2741267e", features = ["blob", "account", "adls_gen2"] }
+azure_core = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev = "536da42ebefd411feff8ba6a0965865e2741267e" }
+azure_storage = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust", optional = true, rev = "536da42ebefd411feff8ba6a0965865e2741267e", features = ["blob", "account", "adls_gen2"] }
```

Comment:
We can leave the azure dependencies as is for now. During releases, we will merge main back into the current stable release branch, which has azure dependencies commented off. This should be good enough to unblock the release :)

Reply:
Understood. Reverted the explicit versions from the azure dependencies.